src/main/java/org/jenkinsci/test/acceptance/po/JenkinsDatabaseSecurityRealm.java

```diff
@@ -81,7 +81,7 @@ public Signup email(String mail) {
 
         public User signup(String name) {
             control(by.input("username")).set(name);
-            control(by.name("Submit")).click();
+            control(by.name("Submit")).clickAndWaitToBecomeStale();
```

Comment:
Wait after submitting the form, similar case to the login (#521)

Reply:
can this use submit?

Reply:
This can be replaced, working on it